src/pages/_app.tsx

```diff
@@ -1,3 +1,5 @@
+import usePendingSafeNotifications from '@/features/counterfactual/hooks/usePendingSafeNotifications'
+import usePendingSafeStatus from '@/features/counterfactual/hooks/usePendingSafeStatuses'
```

Comment:
Ideally feature-specific imports should be lazy.

Reply:
Agree, for now we can lazy-load them only if we detect that there is a counterfactual state similar to how we do it with Recovery hooks.